### ISSUE TITLE
cross encoder classifier

### ISSUE THREAD

**Issue by ReySadeghi** [UKPLab/sentence-transformers#581]
hi,
what is the classifier of cross encoder? is it a softmax layer?

**Comment by nreimers** [comment]
If the number of labels=1, then it is sigmoid + BCE loss. If there are more than 2 labels, then it is softmax + CrossEntropy loss

**Comment by ReySadeghi** [comment]
thanks.
